### PR TITLE
add pyflow

### PR DESCRIPTION
**Pull request by David-OConnor**
[Pyflow](https://github.com/David-OConnor/pyflow) is a Python installation and package manager.

### CODE REVIEW

File: bucket/pyflow.json [hunk 1]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "0.1.5",
+    "homepage": "https://github.com/David-OConnor/pyflow",
+    "description": "A modern Python installation and dependency manager",
```
Comment (Ash258) [inline bucket/pyflow.json]:
```suggestion
    "description": "A modern Python installation and dependency manager.",
```

File: bucket/pyflow.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    "license": "MIT",
+    "url": "https://github.com/David-OConnor/pyflow/releases/download/0.1.5/pyflow.zip",
+    "bin":"pyflow.exe",
+    "checkver": {
```
Comment (Ash258) [inline bucket/pyflow.json]:
```suggestion
    "checkver": "github",
```

File: bucket/pyflow.json [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        "github": "https://github.com/David-OConnor/pyflow"
+    },
+    "autoupdate": {
+        "url": "https://github.com/David-OConnor/pyflow/releases/download/$version/pyflow.zip"
+
```
Comment (Ash258) [inline bucket/pyflow.json]:
```suggestion
```

File: bucket/pyflow.json [hunk 4]
```diff
@@ -0,0 +1,15 @@
+{
+    "version": "0.1.5",
+    "homepage": "https://github.com/David-OConnor/pyflow",
+    "description": "A modern Python installation and dependency manager",
+    "license": "MIT",
+    "url": "https://github.com/David-OConnor/pyflow/releases/download/0.1.5/pyflow.zip",
+    "bin":"pyflow.exe",
```
Comment (Ash258) [inline bucket/pyflow.json]: Missing hash
Use checkver for automatic calculation

File: bucket/pyflow.json [hunk 5]
```diff
@@ -0,0 +1,13 @@
+{
+    "version": "0.1.5",
+    "homepage": "https://github.com/David-OConnor/pyflow",
+    "description": "A modern Python installation and dependency manager.",
+    "license": "MIT",
+    "url": "https://github.com/David-OConnor/pyflow/releases/download/0.1.5/pyflow.zip",
+    "bin": "pyflow.exe",
```
Comment (Ash258) [inline bucket/pyflow.json]:
```suggestion
    "hash": "fc6bad25dc84ba19e887ca7fab4ce6de92be70a366525c14d116eadc7af388b6", 
    "bin": "pyflow.exe",
```

File: bucket/pyflow.json [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+    "hash": "fc6bad25dc84ba19e887ca7fab4ce6de92be70a366525c14d116eadc7af388b6",
```
Comment (Ash258) [inline bucket/pyflow.json]:
```suggestion
```

**Comment by David-OConnor** [comment]
Appreciate the cleanup!

**Pull request merged**